torch/csrc/jit/codegen/cuda/compute_at_map.cpp

```diff
@@ -445,8 +447,8 @@ void ComputeAtMap::build(Fusion* fusion, GpuLower* gpu_lower) {
       int concrete_count = n_concrete_ids_.at(id);
       if (concrete_count >= max_concrete_count) {
         int broadcast_count = n_broadcast_ids_.at(id);
-        if (concrete_count > max_concrete_count ||
-            broadcast_count > max_broadcast_count) {
+        if (concrete_count >= max_concrete_count ||
+            broadcast_count >= max_broadcast_count) {
```

Comment:
IIUC, the concrete ID of the reference tensor is missing a mapping to a root domain in the consumer tensor.

When selecting the concrete ID, the id with the missing root domain has the same `concrete_count` and `broadcast_count`, so it is not selected.

I think this is related to moving the rfactor tensor to a root tensor. The new root ID is counted as a single domain instead of being derived from multiple root domains.

Reply:
@naoyam could you take a look at this please? This seems odd to me.

Reply:
The ViewReductionShmoo test causes the issue.